charts/jenkins/jobs/nuxeo/lts/mirror-nuxeo-2021.groovy.gotmpl

```diff
@@ -15,7 +15,8 @@ pipelineJob('nuxeo/lts/mirror-nuxeo-2021') {
             credentials('github-branch-source')
           }
           // Branches to build
-          branches('*/main')
+          // TODO set back main branch before merging
+          branches('*/task-NXBT-3657-add-lts-2023-pod-templates')
```

Comment:
Once https://github.com/nuxeo/platform-ci-tools/pull/7 is merged, we can set back `main` branch before merging this PR